x-pack/test_serverless/api_integration/test_suites/common/security/api_keys.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const supertest = getService('supertest');
+  let roleMapping: { id: string; name: string; api_key: string; encoded: string };
+
+  describe('security/api_keys', function () {
+    describe('route access', () => {
+      describe('internal', () => {
+        // The create test MUST be executed for the update, get, get all, and invalidate tests to function
+        it('create', async () => {
+          let body: any;
+          let status: number;
+          const requestBody = {
+            name: 'test',
+            metadata: {},
+            role_descriptors: {},
+          };
+
+          ({ body, status } = await supertest
+            .post('/internal/security/api_key')
+            .set(svlCommonApi.getCommonRequestHeader())
+            .send(requestBody));
+          // expect a rejection because we're not using the internal header
+          expect(body).toEqual({
+            statusCode: 400,
+            error: 'Bad Request',
+            message: expect.stringContaining(
+              'method [post] exists but is not available with the current configuration'
+            ),
+          });
+          expect(status).toBe(400);
+
+          ({ body, status } = await supertest
+            .post('/internal/security/api_key')
+            .set(svlCommonApi.getInternalRequestHeader())
+            .send(requestBody));
+          // expect success because we're using the internal header
+          expect(status).toBe(200);
+          roleMapping = body;
+        });
+
+        it('update', async () => {
+          let body: any;
+          let status: number;
+          const requestBody = {
+            id: roleMapping.id,
+            metadata: {},
+            role_descriptors: {},
+          };
+
+          ({ body, status } = await supertest
+            .put('/internal/security/api_key')
+            .set(svlCommonApi.getCommonRequestHeader())
+            .send(requestBody));
+          // expect a rejection because we're not using the internal header
+          expect(body).toEqual({
+            statusCode: 400,
+            error: 'Bad Request',
+            message: expect.stringContaining(
+              'method [put] exists but is not available with the current configuration'
+            ),
+          });
+          expect(status).toBe(400);
+
+          ({ body, status } = await supertest
```

Comment:
__nit:__ not used. Or you planned to assert the body, but forgot?
```suggestion
          ({ status } = await supertest
```